core/controllers/analytics.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Controller for Google Analytics."""
+
+from core.controllers import acl_decorators
+from core.controllers import base
+import feconf
+
+
+class AnalyticsHandler(base.BaseHandler):
```

Comment:
If you need this on every page, is it worth adding it to the BaseHandler and including it as part of the general JSON response?

EDIT: oh, you have a completely separate script for this. I think that's OK, too.